src/Components/BucketForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { push, ref, set } from "firebase/database";
+import { database } from "../firebase/firebase";
+
+const REALTIME_DATABASE_KEY_BUCKET = "bucket-list";
+
+export default function BucketForm({ onSubmit }) {
+  //State for bucketlist
+  const [title, setTitle] = useState("");
+  const [items, setItems] = useState([]);
+  const [newItem, setNewItem] = useState("");
+  const [date, setDate] = useState("");
+  const [isDateSelected, setIsDateSelected] = useState(false);
+
+  //create input to add more items with + button...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    setItems((currentItem) => {
+      return [
+        ...currentItem,
+        {
+          id: Math.floor(Math.random() * 10000),
```

Comment:
Can consider using time to create a unique ID

const timestamp = new Date().getTime();
console.log(timestamp); // Output: 1643501012345